src/classes/CW_CampaignRewardCompControllerTest.cls

```diff
@@ -0,0 +1,260 @@
+/**
+* File Name   :    CW_CampaignRewardCompController.cls
+* Description :    B-109691 - Reward Summary Table 
+*				   that are linked to the campaign 
+*
+* Modification Log
+* ============================================================================
+* Ver Date       Author         Modification
+* --- ---------- -------------- --------------------------
+* 0.1 03/04/2014 Cristina P		Initial
+*/
+@isTest
+public with sharing class CW_CampaignRewardCompControllerTest {
+
+	public static CW_Campaign_Navigation_Settings__c 	cns1;
+	public static CW_Campaign_Navigation_Settings__c 	cns2;
+	
+	
+	public static void initCustomSettings()
+	{
+		cns1 = TestDataUtil.createCampaignNavigationSettings(Page.CW_Campaign_Information,1,false);
+    	cns2 = TestDataUtil.createCampaignNavigationSettings(Page.CW_Audience,2,false);
+    	insert new List<CW_Campaign_Navigation_Settings__c>{cns1,cns2};
+    	
+    	CW_CampaignListSettingHelper.getIntegerValue(ConstantUtil.REWARD_PAGE_SIZE,5);//Set page size to 5 recs
+    	
+	}
+	
+	static testMethod void standardSetControllerTest() 
```

Comment:
Style issue: test methods should be annotated with @IsTest for better readibility